goth_tests/pyproject.toml

```diff
@@ -21,9 +21,9 @@ python = "^3.8.0"
 pytest = "^6.2"
 pytest-asyncio = "^0.20.2"
 pytest-rerunfailures = "^10.3"
-#goth = "^0.11"
+goth = "^0.15.0"
```

Comment:
It is nice outdated Goth release notes got finally updated: https://github.com/golemfactory/goth/commit/a9186a1dc63da49ba75b31a0d286f48fd20bdd08#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR13
It was the main reason we have been using commit hash for so long.